src/main/java/uk/gov/hmcts/ccd/v2/V2.java

```diff
@@ -41,6 +41,8 @@ private MediaType() {}
         public static final String UI_JURISDICTIONS =
             "application/vnd.uk.gov.hmcts.ccd-data-store-api.ui-jurisdictions.v2+json;charset=UTF-8";
 
+        public static final String UI_CASE_EVENTS = "application/vnd.uk.gov.hmcts.ccd-data-store-api.ui-case-events.v2+json;charset=UTF-8";
```

Comment:
If this is to be external then it should not have **ui-** prefix.

Reply:
Changed to external